test/gst-msdk/transcode/transcoder.py

```diff
@@ -69,8 +70,6 @@ def validate_spec(self):
     from slash.utils.pattern_matching import Matcher
 
     assert len(self.outputs), "Invalid test case specification, outputs data empty"
-    if len(self.outputs) > 1 or self.outputs[0].get("channels", 1) > 1:
-      slash.skip_test("transcode 1ton tests not supported")
```

Comment:
We also need to validate all of the outputs (requirements, platforms, mode, etc.) below.  See https://github.com/intel/vaapi-fits/blob/1d853934d6d25fd4164b3339dddae729438feeb2/test/ffmpeg-vaapi/transcode/transcoder.py#L76 as example.

Reply:
sorry, have fixed it.